CONTRIBUTING.md

```diff
@@ -185,7 +185,14 @@ If you are updating tests and just want to run a single test to check it, you
 can use this syntax to run it exactly as the test harness would:
 
 ```text
-$ python tools/test.py -v --mode=release parallel/test-stream2-transform
+$ python tools/test.py --mode=release parallel/test-stream2-transform
```

Comment:
>But that sentence is about running a single test, isn't it?

True

>Is `-J` applicable for folders other than `parallel`?

It is not.

My point is that the purpose of this section to help you run tests as the CI would, and the biggest reason to need to know this is for debugging. Once you know that these are the flags you'll be aware of them, which might make your life substantially easier one day.

Basically I don't see a reason not to include it, and I can see a reason to include it. I could be wrong though, let me know!